### PR TITLE
ref(pkg): remove extract-domain

### LINKED ISSUE #76: modify publish-release-controller to remove the app URL
see deis/workflow#229

### PR DESCRIPTION
**Pull request by bacongobbler**
the domain object is being removed in Deis v2, so this object is
no longer necessary.

closes #76

### CODE REVIEW

**Comment by arschles** [comment]
@bacongobbler assuming this is ready for review, :+1:

**Comment by smothiki** [comment]
this looks good to me . Adding LGTM assuming this is awaiting review .

File: rootfs/etc/confd/templates/builder [hunk 1]
```diff
@@ -185,10 +185,9 @@ if [ $CODE -ne 0 ]; then
 fi
 
 RELEASE=$(echo $PUBLISH_RELEASE | extract-version)
-DOMAIN=$(echo $PUBLISH_RELEASE | extract-domain)
 indent "done, $APP_NAME:v$RELEASE deployed to Deis"
 echo
-indent "http://$DOMAIN"
+indent "Use \`deis open\` to view this application in your browser."
```
Comment (helgi) [inline rootfs/etc/confd/templates/builder]: Maybe do `deis open -a $APP_NAME` ?

Comment (bacongobbler) [inline rootfs/etc/confd/templates/builder]: Typically `-a` isn't necessary as when they're doing a `git push deis master`, they're in the correct directory.

**Pull request merged**